cosmos/dbt/selector.py

```diff
@@ -451,6 +493,31 @@ def select_nodes_ids_by_intersection(self) -> set[str]:
         self.selected_nodes = selected_nodes
         return selected_nodes
 
+    def _should_include_based_on_meta(self, node: DbtNode, config: dict[Any, Any]) -> bool:
+
+        # Deal with meta properties
+        selector_meta_patterns = {key: value for key, value in config.items() if key.startswith(CONFIG_META_PATH)}
+        if selector_meta_patterns:
+            for key, value in selector_meta_patterns.items():
+                if not _check_nested_value_in_dict(node.config, f"{key}:{value}"):
+                    return False
+                config.pop(key)
+        return True
+
+    def _should_include_based_on_config(self, node: DbtNode, config: dict[Any, Any]) -> bool:
```

Comment:
```suggestion
    def _should_include_based_on_non_meta_and_non_tag_config(self, node: DbtNode, config: dict[Any, Any]) -> bool:
```
I know that could be ugly name but wondering if we could call something like that that since meta & tag are still part of config looking from high level, no?